CMakeLists.txt

```diff
@@ -26,6 +26,35 @@ endif()
 
 file(GLOB _files *.fdm_material *.sig)
 
+# Determine version number
+find_package(Git)
+
+set(GIT_VERSION "Unknown")
```

Comment:
I confused this with the version of git you can find. I'd suggest to rename it to something like `REPO_VERSION`.

Reply:
Renamed to PACKAGE_VERSION